app/views/NirmsNumberView.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.LabelSize._
+@import viewmodels.InputWidth._
+
+
+@this(
+        layout: templates.Layout,
+        heading1: components.Heading1,
+        formHelper: FormWithCSRF,
+        govukInput: GovukInput,
+        govukButton: GovukButton,
+        govukErrorSummary: GovukErrorSummary
+)
+
+
+@(form: Form[_])(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle    = titleNoForm(messages("nirmsNumber.title")),
+    showBackLink = true
+) {
+
+
+    @formHelper(action = routes.NirmsNumberController.onSubmit, Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukInput(
+            InputViewModel(
+                field = form("nirmsNumber"),
+                label = LabelViewModel(messages("nirmsNumber.heading")).asPageHeading(Large)
+            ).withHint(Hint(content = Text(messages("nirmsNumber.hint"))))
+            .withWidth(Fixed10)
```

Comment:
Does this limit the number of characters?

Reply:
It just reduces the size of the text field, inline with the prototype. Does not limit the number of characters :)